apps/core/app/[locale]/(default)/cart/_components/shipping-info.tsx

```diff
@@ -217,6 +220,7 @@ export const ShippingInfo = ({
           <FieldLabel htmlFor="zip-field">{t('postcode')}</FieldLabel>
           <FieldControl asChild>
             <Input
+              aria-label={t('postcodeAriaLabel')}
```

Comment:
These should already have an accessible name attributed to them via the [`htmlFor` + `id`](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/for) relationship. These should not be needed.

(Same with the input above)